workflow/usage.rst

```diff
@@ -450,3 +450,215 @@ The following example shows these functions in action:
     {% if 'waiting_some_approval' in workflow_marked_places(post) %}
         <span class="label">PENDING</span>
     {% endif %}
+
+Storing Metadata
+----------------
+
+.. versionadded:: 4.1
```

Comment:
Please add a blank line after the directive

Reply:
Thanks @OskarStark, all included.